arcgis-ios-sdk-samples/Layers/OpenStreetMap layer/OpenStreetMapLayer.storyboard

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="14313.18" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="lIr-n2-Lqu">
+    <device id="retina4_7" orientation="portrait">
+        <adaptation id="fullscreen"/>
+    </device>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="14283.14"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <scenes>
+        <!--Open Street Map Layer View Controller-->
+        <scene sceneID="BhE-do-wJj">
+            <objects>
+                <viewController id="lIr-n2-Lqu" customClass="OpenStreetMapLayerViewController" customModule="arcgis_ios_sdk_samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="0Mi-J9-K9g">
+                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Q3F-3f-v07" customClass="AGSMapView">
+                                <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                                <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                            </view>
+                        </subviews>
+                        <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                        <constraints>
+                            <constraint firstItem="Q3F-3f-v07" firstAttribute="leading" secondItem="0Mi-J9-K9g" secondAttribute="leading" id="2L2-ir-zDf"/>
+                            <constraint firstItem="Q3F-3f-v07" firstAttribute="trailing" secondItem="0Mi-J9-K9g" secondAttribute="trailing" id="VDh-KF-tJ7"/>
+                            <constraint firstItem="Q3F-3f-v07" firstAttribute="bottom" secondItem="0Mi-J9-K9g" secondAttribute="bottom" id="ayY-9E-Cpr"/>
+                            <constraint firstItem="Q3F-3f-v07" firstAttribute="top" secondItem="0Mi-J9-K9g" secondAttribute="top" id="qad-fK-gwx"/>
```

Comment:
Please constrain the top of the map view to the top of the safe area rather than the superview. Constraining to the superview would be nice if we had a translucent navigation bar, but the problem is that the watermark can appear underneath the navigation bar, partially obscuring it.

Reply:
Good catch. I copied the storyboard from the display a map sample so that one should probably get updated at some point.